tests/rbd_mirror/test_rbd_mirror_daemon_restart_imagesync.py

```diff
@@ -0,0 +1,166 @@
+import time
+
+from ceph.parallel import parallel
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+
+
+def run(**kw):
+    """
+    Verification of image sync continuation after multiple rbd mirror daemon restarts.
+    Args:
+        **kw: test data
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    Test case covered -
+    CEPH-10468 - Bug 1348940 - Restart of RBD daemon is again initiating full Sync/Copy of an Image
+    Pre-requisites -
+    Two ceph clusters with rbd mirror configured along with at least 1 client.
+
+    Test Case Flow:
+        1. Configure mirorring between two clusters.
+        2. induce decoy by adding images and io in background
+        3. Reduce concurrent image syncs to delay sync process to be able to verify with focus in
+           images under observation.
+        4. Initiate resync of images and restart rbd mirorring daemon in between.
+        5. Make sure that image sync doesn't start from the begining.
+    """
+    log.info("Starting RBD mirroring test case - CEPH-10468")
+
+    config = kw.get("config")
+
+    if not (config.get("rep_pool_config") or config.get("ec_pool_config")):
+        pool_image_config = {
```

Comment:
Any specific reason you want to specify these values and not take the default ones set by rbd_mirror_config method?

Reply:
1) Wanted to try different combinations, so specified mode, mirror mode
2) Wanted to store pool name and image name for further observatory operations, so specified them
3) since mirror config executes sequential fashion, reduced io and adding io in parallel locally to optimise as initial planned io size was big

Reply:
Okay. LGTM.